Source/ShoppingLists/Mixins/ScrollList.lua

```diff
@@ -83,7 +83,7 @@ end
 
 function AuctionatorScrollListMixin:EndSearch(results)
   self.multiSearchComplete = true
-  C_AuctionHouse.SearchForItemKeys(results, {sortOrder = 1, reverseSort = false})
+  Auctionator.Search.SafeItemKeysSearch(results)
```

Comment:
Curious: Why in the Search namespace rather than in the search provider?

Reply:
We already do C_AuctionHouse.SearchForItemKeys outside the provider. This is just a wrapper for that - not something that affects how the results are procured